.devtools/fop_names_rules.neon

```diff
@@ -0,0 +1,11 @@
+services:
+  -
+    class: FOP\Console\DevTools\PhpStanCustomRule
+    tags:
+      - phpstan.rules.rule
+  -
+    class: FOP\Console\Tests\Validator\FOPCommandFormatsValidator
```

Comment:
this class has to be declared here because it's injected (autowiring) inside the PhpStanNamesConsistencyService.